src/App.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+
+import { TodoList } from "./components/TodoList";
+import TodoForm from "./components/TodoForm";
+
+import "./styles.scss";
+
+export default function App() {
+  const [todos, setTodos] = useState([
+    { id: 1, text: "Buy milk", done: true },
+    { id: 2, text: "Buy bread", done: false },
+  ]);
+
+  function addTodo(todo) {
+    setTodos([
+      ...todos,
+      { ...todo,
+        id: todos.length > 0 ? Math.max(...todos.map(val => val.id)) + 1 : 1 }
```

Comment:
This works, but we're curious why you're mapping over the todos rather than assigning a UUID.